ci/release.js

```diff
@@ -0,0 +1,145 @@
+
+  return changeLog.replace(
+    "## [Unreleased]",
+    `## [Unreleased]\n\n## [${newVersion}] - ${today}`
+  )
+}
+
+const updatePackageJson = (packageJson, version) =>
+  Object.assign({}, packageJson, { version })
+
+const pushCommit = (token, tag) =>
+  git()
+    .addConfig("user.name", "Voyager Bot")
+    .addConfig("user.email", "voyager_bot@tendermint.com")
+    // needed to authenticate properly
+    .addRemote("bot", `https://${token}@github.com/cosmos/voyager.git`)
+    .commit("Bump version for release.", [
+      __dirname + "/../package.json",
+      __dirname + "/../CHANGELOG.md"
+    ])
+    .tag([tag])
+    // The 'push' method doesn't allow defaults so we call it manually.
+    .raw(["push", "bot"])
+
+const publishRelease = (token, tag) =>
+  util.promisify(release)({
+    token,
+    owner: "cosmos",
+    repo: "voyager",
+    tag,
+    name: `Cosmos Voyager Alpha ${tag} (UNSAFE)`,
+    notes: `
+NOTE: DO NOT ENTER YOUR FUNDRAISER SEED. THIS SOFTWARE HAS NOT BEEN AUDITED.
+NEVER ENTER YOUR FUNDRAISER SEED 12 WORDS ONTO AN ONLINE COMPUTER.
+
+Even when we do start supporting fundraiser seeds, don't use it except for
+testing or with small amounts. We will release a CLI to use for offline signing
+of transactions, and we will also add hardware support for this UI.
+
+Please checkout the [CHANGELOG.md](CHANGELOG.md) for a list of changes.
+`,
+    draft: false,
+    prerelease: true,
+    skipAssetsCheck: false,
+    assets: [
+      path.join(__dirname, `../builds/Cosmos Voyager-darwin-x64_${tag}.tar.gz`),
+      path.join(__dirname, `../builds/Cosmos Voyager-linux-x64_${tag}.tar.gz`),
+      path.join(__dirname, `../builds/Cosmos Voyager-win32-x64_${tag}.zip`)
+    ]
+  })
+
+async function main() {
+  console.log(await getBranchName())
+  if ((await getBranchName()) !== "master") {
+    console.log("--- Not releasing. Not a merge to master. ---")
+    return
+  }
+  console.log("Releasing Voyager...")
+
+  const changeLog = fs.readFileSync(__dirname + "/../CHANGELOG.md", "utf8")
+  const packageJson = require(__dirname + "/../package.json")
+  const oldVersion = packageJson.version
+  const newVersion = bumpVersion(oldVersion)
+  const releaseConfig = require(__dirname + "/../release.config.json")
+  const config = toml.parse(
+    fs.readFileSync(__dirname + "/../app/config.toml", "utf8")
+  )
+
+  console.log("New version:", newVersion)
+
+  const newChangeLog = updateChangeLog(changeLog, newVersion, new Date())
+  const newPackageJson = updatePackageJson(packageJson, newVersion)
+
+  fs.writeFileSync(__dirname + "/../CHANGELOG.md", newChangeLog, "utf8")
+
+  fs.writeFileSync(
+    __dirname + "/../package.json",
+    JSON.stringify(newPackageJson, null, 2),
+    "utf8"
+  )
+
+  console.log("--- Committing release changes ---")
+  pushCommit(process.env.GIT_BOT_TOKEN, newVersion)
+  console.log("SDK commit:", releaseConfig.SDK_COMMIT) // TODO put in config.toml?
+
+  if (!process.env.SKIP_BUILD) {
```

Comment:
It doesn't make sense to me to skip the building if we're bumping the version number.  If we don't build, the app will have the old version number.  Is that your intention?

Reply:
Ah sorry, I left that in. I used the flag to quickly test the build script, skipping the long build step.